numcodecs/_utils.pxd

```diff
@@ -0,0 +1,24 @@
+# cython: embedsignature=True
+# cython: profile=False
+# cython: linetrace=False
+# cython: binding=False
+# cython: language_level=3
+
+
+from libc.stdint cimport uint8_t, uint32_t
+
+
+cdef inline void store_le32(uint8_t c[4], uint32_t i) nogil:
```

Comment:
Adding [nogil](url) is good.

Reply:
Yep this forces Cython to generate straight C code. So the resulting code is nearly identical to what was in the header.